### PR TITLE
Move resources to ULID with prefix

### PR DESCRIPTION
**Pull request by nanobash**
## Summary

[Preview](https://preview.redoc.ly/all-rebilly/move-resources-to-ulid%2Fbatch-1/)

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Journal/JournalRecord.yaml [hunk 1]
```diff
@@ -9,7 +9,10 @@ required:
...
 properties:
   id:
-    $ref: ../ResourceId.yaml
+    type: string
+    description: Unique resource ID.
+    maxLength: 50
+    example: jrn_rec_0YVCXV2HE5DBT89QV7RXSJEZQ4
   journalEntryId:
     description: ID of the journal entry.
     readOnly: true
```
Comment (lexerom) [inline openapi/components/schemas/Journal/JournalRecord.yaml:18]: Mb add maxlength and example here too?

Comment (lexerom) [reply openapi/components/schemas/Journal/JournalRecord.yaml:18]: Actually, invoiceId and InvoiceItemId also without examples

Comment (nanobash) [reply openapi/components/schemas/Journal/JournalRecord.yaml:18]: But it is exactly, what was before `$ref: ../ResourceId.yaml`. Even in some examples I am adding `example` if not specified, that might be topic of discussion. But here ResourceId.yaml content is moved and example value is updated, nothing else.

Comment (nanobash) [reply openapi/components/schemas/Journal/JournalRecord.yaml:18]: I'd would not modify other items rather than what's the scope of the PR.

**Review by ashkarpetin — APPROVED** [review]

**Review by lexerom — APPROVED** [review]

**Pull request merged**